src/main/java/org/ow2/proactive/catalogclient/model/CatalogObject.java

```diff
@@ -50,16 +52,14 @@
 
     private final String commitMessage;
 
-    private ImmutableList<CatalogObjectMetadata> metadata;
+    private ImmutableList<CatalogObjectMetadata> objectKeyValues;
 
     private ImmutableList<String> links;
 
-    private String commitDateTime;
+    private String commitTime;
 
-    @Setter
-    @Getter
     @RequiredArgsConstructor
```

Comment:
I can't implement a Builder constructor with CatalogObject as a unique constructor if I use the lombok plugin.